### PR TITLE
Add how to create a tax plugin guide (restaged)

### PR DESCRIPTION
**Pull request by brent-hoover**
A Guide on how to create a plugin that integrates a tax provider.

### CODE REVIEW

**Comment by delagroove** [comment]
@zenweasel the title is the file name.
about the missing section, it's fine for now. we can comment it later.

File: guides/create-a-tax-plugin.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+## At a Glance
+Most companies need to collect and report sales tax, and because of the complexities of understanding
```
Comment (delagroove) [inline guides/create-a-tax-plugin.md]: one extra line break

File: guides/create-a-tax-plugin.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+In general, to add a tax service you must do the following:
+- Create a plugin or modify an existing one
+- Create and register a tax calculation function
+- Optionally create and register a function that returns a list of valid item tax codes
+- If necessary, extend the GraphQL API with mutations, queries, and types needed for operator UI
+- If necessary, create a React component for operators to enter and edit settings for your tax service
+
+## What you need
+- [Understanding how to create and integrate a plugin](/developer/open-commerce/guides/build-api-plugin/)
```
Comment (delagroove) [inline guides/create-a-tax-plugin.md:14]: Extra line break

**Review by delagroove — COMMENTED** [review]
only spacing issues int his one

File: guides/create-a-tax-plugin.md (hunk 1 above)
Comment (brent-hoover) [reply guides/create-a-tax-plugin.md]: Unclear on where the extra line break is

Comment (brent-hoover) [reply guides/create-a-tax-plugin.md]: Tried to adjust this based on what I thought you might have meant. Take a look if that's right

**Review by delagroove — APPROVED** [review]

**Comment by brent-hoover** [comment]
Closing this PR based on new structure plans

**Pull request closed without merging**